DayrSuryan/DSEthiop13.xml

```diff
@@ -48,14 +52,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <msItem xml:id="ms_i1">
                         <locus from="1r" to="32r"/>
                      <title type="incomplete" ref="LIT1979Mashaf"/>
-                        <textLang mainLang="gez"/>
-                        <note> Chants for individual
```

Comment:
If we say "incomplete" in the case of chant mss or anything arranged according to the calendar, t would be helpful to say, somewhere, what is the last feast.

Reply:
I tried, but had trouble. The version you reviewed is the old one. Could you try to look at the complete file that is under "Files changed"?
![image](https://user-images.githubusercontent.com/19650382/80696744-29f96280-8ad8-11ea-995d-1a89319a714c.png)
Thank you :)

Reply:
The title of the work can be arranged as: Mashafa ziq za-Gwandar (catalogued like this in other catalogues, the attribute "of Gwandar" appears in the incipit).

Reply:
In the work record, you mean?

Reply:
Yes. The attribute "of Gwandar" relates to the work.

Reply:
Sorry, but also here, why not "The Book of Ziq of Gwandar"?

Reply:
Very deep reason: because I forgot